server/server.go

```diff
@@ -91,6 +93,35 @@ func (fs *spaFileSystem) Open(name string) (http.File, error) {
 	return f, err
 }
 
+//This function helps in changing base href to point to rootpath as basepath, we are making modification in only server/static/index.html file
+func withrootPath(rootpath string) {
```

Comment:
can you fix the name of the function to follow go format like camelcase

Reply:
fixed the error the naming.